bee-inx/src/block.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_block as bee;
+use inx::proto;
+
+/// The [`Block`] type.
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct Block {
```

Comment:
Below we have `BlockWithMetadata`. Consistently this could be named `BlockWithId` which would more clearly separate it from the `bee_block::Block` type.

Reply:
I'm following the naming conventions from the [protobuf definitions](https://github.com/iotaledger/inx/blob/develop/proto/inx.proto) here.

Reply:
Right...From the proto perspective it might make sense to just name it block (as it doesn't have to care much about the higher layers), but I wonder if that cannot just be abstracted away here at Bee's layer. :shrug: From simply looking at the type definitions in this module (and how similar they are modelled it seems weird, that one is just called `Block` and the other `BlockWithMetadata`. Of course your argument also makes sense, and just follow the naming conventions from upstream. So this is just a suggestion.

Reply:
I think for now, especially since documentation around INX is still sparse, it makes sense to stick with the same naming conventions for now. Once INX is more stable, it might make sense to deviate more from them to make things more Rust-y.